### PR TITLE
ARROW-2293: [JS] Print release vote e-mail template when making source release

### PR DESCRIPTION
**Pull request by TheNeuralBit**

### CODE REVIEW

**Comment by TheNeuralBit** [comment]
@wesm I welcome any suggestions to modify the template, might as well get it right now. The current template is essentially copy-pasted from your 0.3.1 rc0 email

File: dev/release/js-source-release.sh [hunk 1]
```diff
@@ -100,3 +142,61 @@ echo "Success! The release candidate is available here:"
...
+
+This release candidate is based on commit
+${release_hash}
+
+Please download, verify checksums and signatures, run the unit tests, and vote
+on the release. The easiest way is to use the JavaScript-specific release
+verification script dev/release/js-verify-release-candidate.sh.
+
+The vote will be open for at least 24 hours and will close once
+enough PMCs have approved the release.
```
Comment (wesm) [inline dev/release/js-source-release.sh]: We should change the time window in the template to the default "The vote will be open for at least 72 hours". The release manager can change this on an ad hoc basis if we wish to keep doing faster votes while things are moving quickly

**Review by wesm — COMMENTED** [review]
Thanks @TheNeuralBit, just a minor comment. I can try making the next RC with this script

**Review by wesm — APPROVED** [review]
+1

**Comment by wesm** [comment]
thanks @TheNeuralBit!

**Pull request closed without merging**